### DISCUSSION TITLE
Static files removed in production mode on some app errors

### DISCUSSION THREAD

**Discussion by Mehran91z** [vercel/next.js#16387] [Help]
I want to run my project in production mode in CentOS server, When I run `next build && NODE_ENV=production node -r dotenv/config index.js` command, and getting some errors in application, app will be crashed for everyone and all static files in `.next/static` directory will be removed. So we need to run that command again!

**Question**: In witch situation static files in `.next` directory will be removed automatically in production run? and how I can prevent that?

**Comment by timneutkens** [comment]
> p will be crashed for everyone and all static files in .next/static directory will be removed

you likely did not boot up the production server, the production server does not change `.next/static`
